.ci/containers/gcb-terraform-vcr-tester/Dockerfile

```diff
@@ -0,0 +1,11 @@
+from golang:1.16-stretch as resource
+SHELL ["/bin/bash", "-c"]
+# Set up Github SSH cloning.
+RUN ssh-keyscan github.com >> /known_hosts
+RUN echo "UserKnownHostsFile /known_hosts" >> /etc/ssh/ssh_config
+
+RUN apt-get update
+RUN apt-get install -y git jq
```

Comment:
We probably don't need JQ anymore, that was for parsing HTTP responses from teamcity